### DISCUSSION TITLE
Increase z max giving issues

### DISCUSSION THREAD

**Discussion by AAronShambles** [Jyers/Marlin#1845] [General Discussion]
Has anyone successfully extended the z axis on their ENDER 3 V2. I have completed the hardware changes to make mine 450mm high, but when i change the z max in the configuration file then compile it back to a .bin file the printer does not seem to work ie screen rotation is wrong axis don't move ect. Does anyone have a way to send an already compiled .bin file with a z-max of over 400mm? would be very grateful.

**Comment by Sonicboom247** [comment]
> Has anyone successfully extended the z axis on their ENDER 3 V2. I have completed the hardware changes to make mine 450mm high, but when i change the z max in the configuration file then compile it back to a .bin file the printer does not seem to work ie screen rotation is wrong axis don't move ect. Does anyone have a way to send an already compiled .bin file with a z-max of over 400mm? would be very grateful.

can you post the change you made to increase Z to 450?  if you are saying you increased it and when you select Move Z to +450 your Y moves then you might have the wrong firmware or bigger issue
